srtcore/channel.cpp

```diff
@@ -51,7 +51,7 @@ modified by
 *****************************************************************************/
 
 #include "platform_sys.h"
-
+#include <srt.h>
```

Comment:
I'm sorry I didn't see this the previous review round, but this include is likely not required as well.

Reply:
I think I added it because of srt_compat as well, I just removed it